aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move

```diff
@@ -29,6 +29,15 @@ module aptos_framework::aggregator {
         limit: u128,
     }
 
+    /// Represents snapshot of an aggregator at a particular point in time.
+    /// In storage, the value represents the materialized value of the aggregator
+    /// when the snapshot was created.
+    /// During the block execution, the value represents an identifier pointing to the
+    /// Rust `AggregatorSnapshot` which cointains additional details related.
+    struct AggregatorSnapshot has store {
```

Comment:
What about `AggregatorSnapshot` destruction?

Reply:
I added only `store` capability because aggregator only had `store` and is destroyed using a native function `destroy`. 
Are we using `destroy` native function instead of the `drop` capability because we want to store the list of `destroyed_aggregators` in `AggregatorData`?

Reply:
Before we were using destroy because it was a state item. Now we might not need it, as we discussed on Thu. It doesn't matter for this PR, let's just not forget this :)